plugin/trino-hive/src/main/java/io/trino/plugin/hive/orc/OrcReaderConfig.java

```diff
@@ -152,4 +152,19 @@ public OrcReaderConfig setNestedLazy(boolean nestedLazy)
         options = options.withNestedLazy(nestedLazy);
         return this;
     }
+
+    @Deprecated
+    public boolean isShortZoneIdEnabled()
+    {
+        return options.isShortZoneIdEnabled();
+    }
+
+    @Deprecated
+    @Config("hive.orc.short-zone-id-enabled")
```

Comment:
deprecated.hive.orc.short-zone-id-enabled

Reply:
Renamed to `deprecated.hive.read.orc.short-zone-id-enabled`. https://github.com/trinodb/trino/pull/17554#discussion_r1198764931